ingest/annotations.py

```diff
@@ -86,7 +86,15 @@ def preprocess(self):
         # Uppercase NAME and TYPE
         self.headers[0] = self.headers[0].upper()
         self.annot_types[0] = self.annot_types[0].upper()
-        self.create_data_frame()
+        if self.validate_unique_header():
+            self.create_data_frame()
+        else:
+            msg = (
+                "Unable to parse file - Duplicated annotation header names are not allowed. \n"
```

Comment:
"Duplicated" isn't wrong, but "duplicate" seems more idiomatic.  [This thread](https://english.stackexchange.com/q/67463) helped refine my thinking.  The latter is also a bit more internally consistent with the code.

```suggestion
                "Unable to parse file - Duplicate annotation header names are not allowed. \n"
```